app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -192,24 +194,43 @@
 
       this.data.meta = _.map(_.filter(_.pluck(
         iViz.getGroupNdx(this.opts.groupid), this.opts.attrId), function(d) {
-        if (iViz.util.strIsNa(d, true)) {
+        if (typeof d === 'undefined' || d === 'na' || d === '' ||
+          d === 'NaN' || d == null || (isNaN(d) && !d.includes('>') && !d.includes('<'))) {
           d = 'NA';
         }
         return d !== 'NA';
       }), function(d) {
-        var number = parseFloat(d);
-        if (isNaN(number)) {
-          _dataIssue = true;
+        var number = d;
+        if (isNaN(d)) {
+          if (number.includes('<=')) {
```

Comment:
You should think about how to combine these conditions together. A regular expression could be useful with index.